testcafe/tests/helpers/utils.js

```diff
@@ -86,3 +89,30 @@ export async function setDownloadPath(downloadFolderPath) {
     downloadPath: path.resolve(__dirname, downloadFolderPath)
   })
 }
+//Check http status for all img, to be sure everything is loaded before counting images
+export async function checkAllImagesExists() {
+  const images = Selector('img')
+  let count = await images.count
+  let requestsCount = 0
+  let statuses = []
+
+  const getRequestResult = ClientFunction(url => {
+    return new Promise(resolve => {
+      var xhr = new XMLHttpRequest()
+      xhr.open('GET', url)
+      xhr.onload = function() {
+        resolve(xhr.status)
+      }
+      xhr.send(null)
+    })
```

Comment:
Can we use something like this ? 
```
  const getRequestResult = ClientFunction(url => {
  const response = await fetch('GET', url)
  return response.status
 }
```

Reply:
Seems that async/await function won't work inside clientFunction, So I'll still need to use a promise anyway.

Reply:
add async